src/NHSD.BuyingCatalogue.Ordering.OrderingDatabase/Tables/Order.sql

```diff
@@ -1,20 +1,25 @@
 ﻿CREATE TABLE dbo.[Order]
 (
     OrderId NVARCHAR(10) NOT NULL PRIMARY KEY,
-    [Description] NVARCHAR(100) NOT NULL,
+    Description NVARCHAR(100) NOT NULL,
     OrganisationId UNIQUEIDENTIFIER NOT NULL,    
     OrganisationName NVARCHAR(256) NULL, 
     OrganisationOdsCode NVARCHAR(8) NULL, 
     OrganisationAddressId INT NULL,
     OrganisationBillingAddressId INT NULL,    
     OrganisationContactId INT NULL,
     OrderStatusId INT NOT NULL,    
+    SupplierId INT NULL, 

```

Comment:
Really good, would you be able to put this in the Order table (if you have a look at the latest PR, I've setup foreign keys using the data annotations, as otherwise this broke the integration tests as EF was struggling to map

Reply:
Not sure what you are saying here do you want me to add the attributes to the data model?

Reply:
To the Order.cs (in the domain)

Reply:
This is an example of linking the foreign key to it

Reply:
[ForeignKey("OrganisationAddressId")]
        public Address OrganisationAddress { get; set; }

From https://github.com/nhs-digital-gp-it-futures/BuyingCatalogueOrdering/pull/48/files

Reply:
ok your comment said order table , but you i think you mean i should add he foreign key attributes to the orders model? ,  also SupplierId does not map to a table row at the moment so i guess you mean just  the attributes for the other relations  address and contact?

Reply:
Yes sorry, use the foreign key attribute on address and contact, but also include the supplier name and supplierId on the order.cs class

Reply:
I need to patch tests if i do model stuff now, i kind of think the model stuff should have been done in a separate commit .  now the column commit will have model changes and test changes too.

Reply:
Going to do the model changes in a separate commit